packages/vuetify/src/components/VDatePicker/VDatePicker.ts

```diff
@@ -121,6 +121,7 @@ export default mixins(
       type: String,
       default: '$vuetify.datePicker.itemsSelected',
     },
+    showSiblingMonths: Boolean,
```

Comment:
`showAdjacentMonths`? "sibling" implies possibly not directly next to.

Reply:
Or just have it be default behaviour.

Reply:
Same question as for hold-on-hover for snackbar - do we need a prop to turn this behaviour off?

Reply:
I don't think so.